### DISCUSSION TITLE
Questions about Sequencer Node

### DISCUSSION THREAD

**Discussion by Ezra-Labs** [ethereum-optimism/developers#473] [General]
1) What specifically is a sequencer doing that other nodes are not?
2) If there are two nodes, a sequencer and non-sequencer, and the sequencer goes offline, does the L2 continue producing blocks and sending transactions to L1?
3) If the sequencer then returned online, would it sync rapidly back to current state due to second node existing, or would there be another long drawn out process to bring everything back to current state with ETH mainnet?

BONUS question: When I start up a local node on a fresh rollup testnet, and then go offline, and then come back online, it takes the proposer a very long time to catch up, depending on time offline, and this makes testing new contracts locally a chore, due to time contraints. If possible, I would like to not keep processes running continuously, if avoidable, but will do what I have to do. Is there a way to tweak proposer config to more rapidly sync to current state?

**Comment by Ezra-Labs** [comment]
I was able to conclude answers based on info provided here on github. It sounds like if there are 100 nodes but no sequencer node, then no transactions will be sent to eth mainnet?

Is it beneficial to have more than one sequencer running? Do they interfere with one another or behave redundantly?

**Reply by Jonney** [reply]
Based on my operational experience, if a second node runs the sequencer, this node will also generate new unsafe blocks. However, it will reorg, which means it's forked.

**Reply by Ezra-Labs** [reply]
This is really helpful, thanks
